.circleci/config.yml

```diff
@@ -46,3 +46,18 @@ jobs: # a collection of steps
       - store_test_results: # Upload test results for display in Test Summary: https://circleci.com/docs/2.0/collect-test-data/
           path: test_results
       # See https://circleci.com/docs/2.0/deployment-integrations/ for example deploy configs
+workflows:
```

Comment:
Ahh I see, it looks like we still needed to include the `build-test-deploy` component so we could run `build-and-test`, because it's not doing anything at all without this instruction. I can also see that CI is up and running on this PR, so pretty sure it worked??

Reply:
Approving